assets/css/_index_component.scss

```diff
@@ -0,0 +1,81 @@
+%bi-icon-before {
+  display: inline-block;
+  font-family: bootstrap-icons !important;
+  font-style: normal;
+  font-weight: normal !important;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  vertical-align: text-bottom;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+%bi-check-circle-fill-before {
+  &::before {
+    content: "\f26a";
+  }
+}
+
+%bi-plus-circle-fill-before {
+  &::before {
+    content: "\f4d2";
+  }
+}
+
+%bi-circle-fill-before {
+  &::before {
+    content: "\f287";
+  }
+}
+
+%bi-x-circle-fill-before {
+  &::before {
+    content: "\f5e6";
+  }
+}
+
+.IndexComponent__IdeaLabel {
```

Comment:
i would use a normal class for this, why the IndexComponent part?

Reply:
We would like to use the BEM css naming convention to avoid css collisions, see http://getbem.com/naming/